script/tool/CHANGELOG.md

```diff
@@ -16,7 +20,7 @@
 - Adds a new `update-excerpts` command to maintain README files using the
   `code-excerpter` package from flutter/site-shared.
 - `license-check` now ignores submodules.
-- Allows `make-deps-path-based` to skip packages it has alredy rewritten, so
+- Allows `make-deps-path-based` to skip packages it has already rewritten, so
```

Comment:
`to skip packages it has already rewritten` seems to be grammatically incorrect.